driver/main.c

```diff
@@ -955,24 +955,12 @@ static long ppm_ioctl(struct file *filp, unsigned int cmd, unsigned long arg)
 	switch (cmd) {
 	case PPM_IOCTL_DISABLE_DROPPING_MODE:
 	{
-		struct event_data_t event_data;
-
 		vpr_info("PPM_IOCTL_DISABLE_DROPPING_MODE, consumer %p\n", consumer_id);
 
 		consumer->dropping_mode = 0;
 		consumer->sampling_interval = 1000000000;
 		consumer->sampling_ratio = 1;
 
-		/*
-		 * Push an event into the ring buffer so that the user can know that dropping
-		 * mode has been disabled
-		 */
-		event_data.category = PPMC_CONTEXT_SWITCH;
-		event_data.event_info.context_data.sched_prev = (void *)DEI_DISABLE_DROPPING;
```

Comment:
We can drop the `DEI_DISABLE_DROPPING` constant now (and the whole `ppm_driver_event_id` enum)